receiver/githubactionsreceiver/trace_event_handling.go

```diff
@@ -168,6 +168,12 @@ func createSpan(scopeSpans ptrace.ScopeSpans, step *github.TaskStep, job *github
 	span.SetParentSpanID(parentSpanID)
 
 	var spanID pcommon.SpanID
+	
+	if job.GetHeadBranch() == "main" || job.GetHeadBranch() == "master" {
```

Comment:
Would it be possible to use [`default_branch`](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#get-a-repository) to avoid hardcoding this?